config/config.example.yml

```diff
@@ -15,6 +15,93 @@ db:
   port: 5432
   dbname: invidious
 
+
+  ## -----------------------------------
+  ## Database connection pool settings
+  ## -----------------------------------
+
+  ##
+  ## The maximum allowed number of connections within the connection pool.
+  ## When this number is reached and no connections are free, Invidious will
+  ## wait for up to `checkout_timeout` before raising an exception.
+  ##
+  ## Note: Setting to 0 will disable this limit and allow the the pool to
+  ## grow indefinitely.
+  ##
+  ## Accepted values: a positive integer
+  ## Default: 100
+  ##
+  #max_pool_size: 100
```

Comment:
This is a probably a non issue but

Although all of this is nested under the db field, the options here are still really similar to the ones for the HTTP connection pool which might get confusing. And this could be made worse if we ever decide to offer more granular control for the latter (eg configuring the pool options for each domain) which will lead to a lot of essentially duplicate options.